src/main/resources/com/xebialabs/xlt/ci/XLTestView/config.jelly

```diff
@@ -32,6 +32,9 @@
                     <f:entry field="excludes" title="${%Exclude pattern}">
                         <f:textbox default=""/>
                     </f:entry>
+                    <f:entry title="${%XL TestView can make build unstable}">
```

Comment:
Asked Amy, maybe better to say 'Allow XL TestView to set the build to unstable'.